integration/js/steps/OpenAppStep.js

```diff
@@ -17,6 +17,7 @@ class OpenAppStep extends AppTestStep {
   async run() {
     this.logger(`Opening : ${this.url}`);
     await this.page.open(this);
+    await this.page.waitForPageToLoad();
```

Comment:
If `waitForPageToLoad` is only needed for mobile testing then will it add a delay for integration tests as well? I think not but want to know if you have tested that.

Reply:
Why do we need to do this? Is this something specific to iOS?

Reply:
This technically will add a very slight delay to all tests by having to call the method and go through the first few steps but it won't actually do any waiting unless the page hasn't been loaded for some reason which in that case we want to wait a moment. At the moment this is a IOS Safari specific problem, I'm not entirely sure why the open function returns before the page is actually ready but I've even tried calling selenium driver waitforpage and it says it's completed before it actually finishes.